.gitlab-ci.yml

```diff
@@ -23,8 +23,10 @@ init:
 
 check-build-coverage:
   stage: verify
-  image: fedora:latest
+  extends: .terraform
   variables:
+    RUNNER: aws/fedora-40-{arch}
```

Comment:
This doesn't work :/ Do we actually to have this parametrized?

Reply:
ugh, generally yes for the build jobs, which I copy+pasted from, but for this, we need a fixed arch.  SORRY!